index.css

```diff
@@ -0,0 +1,17 @@
+* {
+  width: auto;
+  height: auto;
+  box-sizing: border-box;
+}
```

Comment:
- [ ] Please follow the css best practice. Remove width and height and replace with padding and margin.